gearbox/gearbox.go

```diff
@@ -0,0 +1,32 @@
+package gearbox
+
+import "fmt"
+
+const (
+	first = iota + 1
+	second
+	third
+	fourth
+)
+
+type Gearbox struct {
+	Gear int
+}
+
+type GearboxManipulator interface {
+	SetGear(int)
+	GetGear() int
+	Print()
+}
+
+func (g *Gearbox) SetGear(gear int) {
```

Comment:
probably want validation here if intending to use the consts from above.

Reply:
I finally decided not to use them. d9e86fabc6001384eb73051664cc5a0d5a47cf6f